tests/unit/components/QueryBuilder.spec.ts

```diff
@@ -111,4 +111,30 @@ describe( 'QueryBuilder.vue', () => {
 		} );
 
 	} );
+
+	it( 'should not have obvious accessibility issues', async () => {
```

Comment:
could we move this out of the unit tests and into its own `a11y` test directory that is run on `npm run test:a11y` ?

Reply:
Done.